bitnami/fluentd/README.md

```diff
@@ -299,6 +308,52 @@ aggregator.extraEnv[1].name=ELASTICSEARCH_PORT
 aggregator.extraEnv[1].value=your-port-here
 ```
 
+### Forwarder Security Context & Policy
+
+By default, the **forwarder** `DaemonSet` from this chart **runs as the `root` user**, within the `root` group, assigning `root` file system
+permissions. This is different to the default behaviour of most Bitnami Helm charts where we
+ [prefer to work with non-root containers](https://docs.bitnami.com/tutorials/work-with-non-root-containers/).
+
+The default behaviour is to run as `root` because
+- the forwarder needs to mount `hostPath` volumes from the underlying node to read Docker container (& potentially other) logs
+- in many Kubernetes node distributions, these log files are not readable by anyone other than `root`
+- `fsGroup` doesn't work with `hostPath` volumes to allow the process to run non-root with alternate file system permissions
+
+Since we would like the chart to work out-of-the-box for as many users as possible, the `forwarder` thus runs as root 
+by default. You can read more about the motivation for this at [#1905](https://github.com/bitnami/charts/issues/1905) 
+and [#2323](https://github.com/bitnami/charts/pull/2323), however you should be aware of this, and the risks of running
+root containers in general.
```

Comment:
Why a new line here?

Reply:
Yeah, possibly my bad - I use a loose (i.e inconsistent - eek) 120 char line-wrap indicator in my IDE (except for tables) to improve readability when editing the markdown.

Is there a particular guide for line wrap to use? Perhaps you don't add new-line wraps at all if I am to infer from the existing format?

Reply:
Yes, we don't apply a new-line format by now, we add it when actually it is a new-line.

Reply:
OK, no problems. Fixed now.